web/source/osk/oskKey.ts

```diff
@@ -483,13 +483,30 @@ namespace com.keyman.osk {
       return t;
     }
 
-    public isUnderTouch(touch: Touch): boolean {
-      let x = touch.clientX;
-      let y = touch.clientY;
+    public isUnderTouch(input: InputEventCoordinate): boolean {
+      let x = input.x;
+      let y = input.y;
 
       let btn = this.btn;
+      // These functions do not account for 'fixed' positioning.
```

Comment:
Is this likely to be an issue elsewhere? Do we need to abstract the `isFixed` test into a new version of `getAbsoluteX/Y` (`getAbsolutePosition(elem, testFixed: boolean) returns {x:, y:}`)?

Reply:
Not as far as the OSK is concerned; I remember double-checking for that aspect.  The change _here_ is motivated by the paradigm enforced by the new `InputEventCoordinate` type, as it explicitly uses `pageX` positioning.  `pageX` is affected by the page's _scroll_, while `clientX` positioning (used before these changes) is not.

Now, before this and its followup PR, `isUnderTouch` only ever had to operate for touch keyboards, all of which have always used `fixed` positioning.  The issue is that our _desktop_ OSKs tend to use `absolute` instead, which behaves differently from `fixed` when the page is scrolled.  These two PRs are all about merging the codepaths to work with both OSK types, so this is when the issue arose.

------

Modifying them is also fairly dangerous - they're "unofficial" APIs used by our desktop UI modules.  (I guess that's why the _new version_ suggestion.)

They're _also_ used by both of the longpress codepaths, both of the key-preview codepaths, and by the embedded mode's globe-key logic, to name a few things.  While the test suite specified for this PR didn't explicitly double-check globe key use, it _did_ test subkey behavior; said subkey behavior should remain as it is.  Such tests would have also passively tested key previews.